src/serve/server.js

```diff
@@ -33,6 +34,24 @@ const [assetsFilepath, renderFilepath] = checkBuildArtifacts(
 const assets = processAssets(assetsFilepath);
 const render = require(renderFilepath).default; // eslint-disable-line import/no-dynamic-require
 
+let ERROR_MESSAGE = '';
+const errorHandler = (err, req, reply) => {
+  if (!ERROR_MESSAGE) {
+    try {
+      ERROR_MESSAGE = fs.readFileSync(path.join(paths.public.root, '500.html'), 'utf-8');
+    } catch (missingFileErr) {
+      ERROR_MESSAGE = 'Something went wrong, please try again!';
+    }
+  }
+
+  console.error(err.message); // eslint-disable-line no-console
```

Comment:
@rucsi is this enough for tracking the error, or do we need the entire error object?